ck.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/sh
+
+set -e
+
+function init() {
+	# Make sure submodule is registered and up-to-date.
+	git submodule update --init
+
+	# Fetch remote changes.
+	cd ckeditor-dev
+	# not needed git fetch
+
+	# Make sure our working copy is clean.
+	git reset --hard HEAD --quiet
+	git clean -fdx
+}
+
+function message() {
+	if [[ ! -z "$1" ]] ; then
+		echo
+		printf -- '-%.0s' {1..80}; echo ""
+		printf "%*s\n" $(( ( $(echo $1 | wc -c ) + 80 ) / 2 )) "$1"
```

Comment:
#fancier!

It wasn't obvious to me that this right-aligns. You could make it obvious by pulling it into a function.

```sh
function right_justify() {
  # ...
}
```